### DISCUSSION TITLE
Get Positron release + build info from CLI

### DISCUSSION THREAD

**Discussion by connortwiegand** [posit-dev/positron#4325] [Q&A] [answered]
Is there a way to see the positron release + build version from the CLI? I just want to see some of the info that's available from `Help > About` (from within Positron) without opening Positron. I'm working in Bash on Ubuntu. Current output of `positron -v` only produces the `code` version with a commit ID and an x64 tag. 

Ex: `Positron Version: 2024.07.0 build 125`

**Comment by jthomasmock** [comment, marked as answer]
Thanks for question -- I would follow: https://github.com/posit-dev/positron/issues/1373

**Reply by connortwiegand** [reply]
Thanks!
